### PR TITLE
Vue migration completed for the web app

### PR DESCRIPTION
**Pull request by Lakshya2610**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -6,60 +6,6 @@ import TextField from "material-ui/TextField";
 import io from "socket.io-client";
 import { MuiThemeProvider, createMuiTheme } from 'material-ui/styles';
 
-const ENDPOINT = "http://toonin-backend-54633158.us-east-1.elb.amazonaws.com:8100/";
```
Comment (ArsalaBangash) [inline app/src/App.js]: We should probably revert this file

File: client/server.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const express = require('express')
```
Comment (ArsalaBangash) [inline client/server.js]: Is this file needed to get your client running on AWS?

File: client/src/app.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+var socket;
+var audioElem;
+
+export function init(audioElement) {
+    audioElem = audioElement;
+    setSocketListeners = setSocketListeners.bind(this);
+    createAnswer = createAnswer.bind(this);
+}
+
+export function toonin() {
```
Comment (ArsalaBangash) [inline client/src/app.js]: Can we get rid of this function?

File: client/src/app.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    return;
+}
+
+export function logMessage(msg) { console.log(msg); }
+
+export function checkstream() {
+    var obj = {
```
Comment (ArsalaBangash) [inline client/src/app.js]: Could be renames to state and made a const variable

File: client/src/app.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    createAnswer = createAnswer.bind(this);
+}
+
+export function toonin() {
+    return;
+}
+
+export function logMessage(msg) { console.log(msg); }
+
+export function checkstream() {
```
Comment (ArsalaBangash) [inline client/src/app.js]: A better name for this function could be checkRoom()

File: client/src/app.js [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+            this.room = obj.room,
+            this.isPlaying = obj.isPlaying,
+            this.established = obj.established,
+            this.rtcConn = obj.rtcConn,
+            this.peerID= obj.peerID
+        })
+        .catch(err => logMessage(err));
+}
+
+export function checkStreamResult(result, obj) {
```
Comment (ArsalaBangash) [inline client/src/app.js]: We could split this function into onRoomFound and onNoSuchRoom. Also modified the passed in obj isn't super ideal.

File: client/src/app.js (hunk 4 above)
Comment (ArsalaBangash) [reply client/src/app.js]: In fact, we may want to refactor this state to Vuex so that we could keep track of where we are in the connection process and update it on the UI

File: app/src/App.js (hunk 1 above)
Comment (Lakshya2610) [reply app/src/App.js]: Yeah, the changes on this file are irrelevant. They can be discarded safely

File: client/server.js (hunk 2 above)
Comment (Lakshya2610) [reply client/server.js]: Yes. This is necessary to run the production build on AWS instances

File: client/src/app.js (hunk 3 above)
Comment (Lakshya2610) [reply client/src/app.js]: No, because it sets the audioElem variable to hold reference to the audio tag in our HTML file. This is necessary to attach the received stream to the audio tag

File: client/src/app.js (hunk 6 above)
Comment (Lakshya2610) [reply client/src/app.js]: Yeah, there's a lot to change, especially with the obj. I just hacked some stuff together to get it running. It needs some refactoring
